doc/source/guide/data_types/maps.rst

```diff
@@ -137,28 +151,330 @@ more complicated figure.
    is not met, when the map is plotted a warning will be issued. You can create
    an oriented map by using `~sunpy.map.GenericMap.rotate()` before you plot the Map.
 
-6. Overlaying Maps
-------------------
-The `Map()` method described above can also handle a list of maps. If a list in inputs
-is supplied, `Map()` will return a list of maps as the output.  However, if the
-'composite' keyword is set to True, then a `CompositeMap` object is returned.  This is useful if the maps are
-of a different type (e.g. different instruments).  For example, to create a simple composite map::
+7. Plotting Keywords
+********************
+
+For Map `~matplotlib.pyplot.imshow` does most of the heavy
+lifting in the background while SunPy makes a number of choices for you so that
+you don't have to (e.g. colortable, plot title). Changing these defaults
+is made possible through two simple interfaces. You can pass any
+`~matplotlib.pyplot.imshow` keyword into
+the plot command to override the defaults for that particular plot. The following
+plot changes the default AIA color table to use an inverse Grey color table.
+
+.. plot::
+
+    import sunpy.map
+    import sunpy.data.sample
+    import matplotlib.pyplot as plt
+    smap = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+    fig = plt.figure()
+    smap.plot(cmap=plt.cm.Greys_r)
+    plt.colorbar()
+    plt.show()
+
+You can view or make changes to the default settings through the `~sunpy.map.GenericMap.plot_settings`
+property. In the following example we change the title of the plot by changing the
```

Comment:
replace: the `~sunpy.map.GenericMap.plot_settings` property.

with: the `~sunpy.map.GenericMap.plot_settings` dictionary.